frontend/src/app/features/boards/board/inline-add/board-inline-add-autocompleter.sass

```diff
@@ -1,4 +1,7 @@
 ng-select.wp-inline-create--reference-autocompleter
+  position: absolute
+  width: 500px
+  z-index: 999
```

Comment:
Instead of increasing the whole autocompletet, I'd suggest something like this:

```
.ng-dropdown-panel.wp-inline-create--reference-autocompleter
  max-width: 75vw !important
  width: 500px !important
```